rstudio.Rproj

```diff
@@ -7,6 +7,7 @@ AlwaysSaveHistory: No
 EnableCodeIndexing: Yes
 UseSpacesForTab: Yes
 NumSpacesForTab: 3
+UseNativePipeOperator: Never
```

Comment:
btw, I've been wondering the same thing about `PackageCleanBeforeInstall: Yes`

Reply:
Yes, thanks for the suggestion! You absolutely can have implicit options that are only written to the RProj file if other than the default. I've now made the default "(Default)" as in, inherit the global setting, and if so it will not be written to the RProj file